### DISCUSSION TITLE
Whether the call method of the conrtolnet/txt2img API has changed？

### DISCUSSION THREAD

**Discussion by wym19970515** [Mikubill/sd-webui-controlnet#368] [Q&A]
After I pulled the latest code, the conrtolnet/txt2img interface did not work properly. After calling the API, no line draft was generated, and the effect fell back to the original txt2img. This problem was solved after I rolled back the code to the "fix: move leres model to correct device (https://github.com/Mikubill/sd-webui-controlnet/issues/293)" version.

**Comment by Mikubill** [comment]
see https://github.com/Mikubill/sd-webui-controlnet/pull/194#issuecomment-1445051668

**Reply by wym19970515** [reply]
Thanks
